### PR TITLE
Allow user to specify a sha / pull request

### PR DESCRIPTION
**Pull request by blckmn**

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
- How can we test this?
- Can we make this into another input field?
- Please add an input field for adding (testing) #defines.

**Review by haslinghuis — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**
